sidecar/config.go

```diff
@@ -108,9 +149,62 @@ func NewConfig() *Config {
 
 		AdmitDefeatHearbeatCount: int32(admitDefeatHearbeatCount),
 		ElectionTimeout:          int32(electionTimeout),
+
+		ClusterName:                getEnvValue("SERVICE_NAME"),
+		BackupUser:                 "sys_backups", //getEnvValue("BACKUP_USER"),
```

Comment:
`BackupUser` and `BackupPassword` cannot directly specify, maybe you can use OperatorUser and OperatorPassword as the backup user and password